### DISCUSSION TITLE
how i can change commands

### DISCUSSION THREAD

**Discussion by hte4491** [jagrosh/MusicBot#1127] [Questions]
I want change bot commands because @botnametag play song name is hard for some people so i want change bot commands how i can?

**Comment by YJSoft** [comment]
https://jmusicbot.com/config/

```
// These settings allow you to configure custom aliases for all commands.
// Multiple aliases may be given, separated by commas.
//
// Example 1: Giving command "play" the alias "p":
// play = [ p ]
//
// Example 2: Giving command "search" the aliases "yts" and "find":
// search = [ yts, find ]

aliases {
  // General commands
  settings = [ status ]

  // Music commands
  lyrics = []
  nowplaying = [ np, current ]
  play = []
  playlists = [ pls ]
  queue = [ list ]
  remove = [ delete ]
  scsearch = []
  search = [ ytsearch ]
  shuffle = []
  skip = [ voteskip ]

  // Admin commands
  prefix = [ setprefix ]
  setdj = []
  settc = []
  setvc = []

  // DJ Commands
  forceremove = [ forcedelete, modremove, moddelete ]
  forceskip = [ modskip ]
  movetrack = [ move ]
  pause = []
  playnext = []
  repeat = []
  skipto = [ jumpto ]
  stop = []
  volume = [ vol ]
}
```

You can set command alias. (like `!!p` as `!!play`)

p.s.) default command for play is `!!play`, not `@botname play`.)

**Reply by lilHinhbongems** [reply]
@YJSoft how can I custom the prefix ?

**Reply by MichailiK** [reply]
The [config.txt](https://jmusicbot.com/config/) has a line where you can specify the prefix.

![image](https://github.com/jagrosh/MusicBot/assets/39029839/e6707c9c-2488-4d4a-850b-1a148f84f07c)

**Reply by alexysarts** [reply]
It's letting me set a prefix, but it's not letting me change the commands. For example, when I try to change the queue command to q ...it's not doing anything.
